extensions/src/ACRE2Core/NamedPipeServer.h

```diff
@@ -22,33 +22,58 @@ class CNamedPipeServer : public IServer, public CLockable {
     ACRE_RESULT initialize( void );
     ACRE_RESULT shutdown( void );
 
-    ACRE_RESULT handleMessage(unsigned char *data) { data=data;return ACRE_NOT_IMPL; }
+    ACRE_RESULT handleMessage(unsigned char *data) { data=data; return ACRE_NOT_IMPL; }
 
     ACRE_RESULT sendMessage( IMessage *message );
 
     ACRE_RESULT release( void ) { return ACRE_OK; };
-    
+
     ACRE_RESULT checkServer( void ); // DRM
 
     char *currentServerId;
 
-    DECLARE_MEMBER(HANDLE, PipeHandleRead);
-    DECLARE_MEMBER(HANDLE, PipeHandleWrite);
-    DECLARE_MEMBER(BOOL, ConnectedWrite);
-    DECLARE_MEMBER(BOOL, ConnectedRead)
-    DECLARE_MEMBER(ACRE_ID, Id);
-    DECLARE_MEMBER(BOOL, ShuttingDown);
-    DECLARE_MEMBER(std::string, FromPipeName);
-    DECLARE_MEMBER(std::string, ToPipeName);
+    bool getConnected() const { return (this->getConnectedRead() && this->getConnectedWrite()); };
+    void setConnected (const bool value) { this->setConnectedRead(value); this->setConnectedWrite(value); };
+
+    virtual __inline void setPipeHandleRead(const HANDLE value) { this->m_pipeHandleRead = value; }
```

Comment:
These functions should be `override`, not `virtual`
```suggestion
    void setPipeHandleRead(const HANDLE value) override { this->m_pipeHandleRead = value; }
```